### DISCUSSION TITLE
Adding tor to the package

### DISCUSSION THREAD

**Discussion by ghost** [ajax-lives/NoRussian#29] [Ideas]
Isn't it more secure to add tor to the attack?

Because of that I added some content to this package so that it can now use tor.
Added "LaunchTorBrowser.bat" and "Download TorBrowser".

Could you guys maybe add this to your package? Thanks!

[NoRussian.zip](https://github.com/ajax-lives/NoRussian/files/8153986/NoRussian.zip)

**Comment by thibaultmol** [comment]
So your plan is to overload the already overloaded tor network?

(Which anti-Putin Russians are using to get behind the website blocks
they're doing by the gov.)

**Reply by ghost** [reply]
So your advice is to use clearned, isn't it?

**Comment by thibaultmol** [comment]
Use what?

All I'm saying is that running a ddos attack via tor is a bad idea. Tor is
probably being used on both sides. But all that would do is congest an
already congested tor network

**Reply by ghost** [reply]
Well, i aint got no mood to be hunted from some russian agents or the goverment, because I used clearnet. But in general I think same as you. There is to many traffic going on on tor.
